inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/workmanager/schedulers/MessageMixerPingScheduler.kt

```diff
@@ -27,6 +27,7 @@ internal interface MessageMixerPingScheduler {
      */
     fun pingMessageMixerService(initialDelay: Long, workManager: WorkManager? = null)
 
+    @SuppressWarnings("kotlin:S6515")
```

Comment:
BTW, May I know what is this `S6516` warning ?
- It seems to be there for every `companion object` ?

Reply:
https://rules.sonarsource.com/kotlin/RSPEC-6515/.
Encourages to use the built-in feature `object`, instead of manually creating singletons.